src/plugins/wizard/public/embeddable/wizard_embeddable.tsx

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Subscription } from 'rxjs';
+
+import { WizardSavedObjectAttributes } from '../../common';
+import {
+  Embeddable,
+  EmbeddableOutput,
+  IContainer,
+  SavedObjectEmbeddableInput,
+} from '../../../embeddable/public';
+import { IToasts, SavedObjectsClientContract } from '../../../../core/public';
+import { WizardEmbeddableComponent } from './wizard_component';
+import { ReactExpressionRendererType } from '../../../expressions/public';
+import { TypeServiceStart } from '../services/type_service';
+import { DataPublicPluginStart } from '../../../data/public';
+
+export const WIZARD_EMBEDDABLE = 'WIZARD_EMBEDDABLE';
+
+// TODO: remove search, hasMatch or update as appropriate
```

Comment:
Do you have a good understanding of how the inputs and outputs of embeddable work? What are the inputs we can expect from an embeddable? and how do/should they affect the visualization.

I assume this has something to do with the search bar, timerange and filters and how they affect the visualization.

Reply:
Yeah, this will be removed - it's basically some leftover stuff from the embeddable example, and tracked in https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1910

Reply:
removed in https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1935